doc/help.md

```diff
@@ -1,70 +1,205 @@
 # Help
 
-## Publishing a package
+The following help pages are available:
 
-[Pub][pub-tool] isn't just for using other people's packages.
-It also allows you to share your packages with the world.
-If you have a useful project and you want others to be able to use it,
-use the [`pub publish`][pub-publish] command.
+ - [Search on pub.dev](/help/search)
+ - [Publishing packages](/help/publishing)
+ - [Package scores & pub points](/help/scoring)
 
-## Search
+# Search
 
 `pub.dev` supports the following search expressions:
-  - `"exact phrase"`: By default, when you perform a search, the results
-    include packages with similar phrases. When a phrase is inside quotes,
-    you'll see only those packages that contain exactly the specified
-    phrase.
-  - `package:prefix`: Searches for packages that begin with `prefix`.
-    Use this feature to find packages in the same framework.
+
+  - `"exact phrase"`: By default, when you perform a search, the results include
+    packages with similar phrases. When a phrase is inside quotes, you'll see
+    only those packages that contain exactly the specified phrase.
+
+  - `package:prefix`: Searches for packages that begin with `prefix`. Use this
+    feature to find packages in the same framework.
+
   - `dependency:package_name`: Searches for packages that reference
     `package_name` in their `pubspec.yaml`.
+
   - `dependency*:package_name`: Searches for packages that depend on
     `package_name` (as direct, dev, or transitive dependencies).
-  - `email:user@example.com`: Search for packages where either the
-    author or the uploader has the specified email address.
 
-## Scoring
+  - `email:user@example.com`: Search for packages where either the author or the
+    uploader has the specified email address.
+
+## Search filters
+
+The search UI also supports filters:
+
+  - On the left of the search bar, select 'Dart' or 'Flutter' depending on the
+    SDK you are developing with.
+
+  - On the right of the search bar, click the platform filter buttons to filter
+    results to packages supporting the platforms you need support for.
+
+  - To use platform filters for non-stable platforms (e.g. Flutter Desktop), use
+    the Advanced option on the far right of the search bar.
```

Comment:
```suggestion
  - To use platform filters for prerelease platforms (e.g. Flutter Desktop), use
    the Advanced option on the far right of the search bar.
```

Reply:
I don't think Flutter uses that terminology. Changed to alpha platforms.